config/cosmosdb/config.go

```diff
@@ -40,6 +45,14 @@ func Configure(p *config.Provider) {
 			},
 		}
 		r.UseAsync = true
+		r.ExternalName = config.NameAsIdentifier
+		r.ExternalName.GetExternalNameFn = common.GetNameFromFullyQualifiedID
+		// /subscriptions/000-000/resourceGroups/rg1/providers/Microsoft.DocumentDB/databaseAccounts/acc1/sqlDatabases/db1/containers/container1
+		r.ExternalName.GetIDFn = common.GetFullyQualifiedIDFn("Microsoft.DocumentDB",
```

Comment:
Are we sure we don't want the group of this resource should be `cosmosdb` instead of `documentdb`?

Reply:
Not sure either. But I believe (though definitely not sure) that `cosmodb` is a better known name. Also the Terraform counterparts have this prefix.